Fix 4367 move client library starter (#4380)
influxdata/docs-v2: content/influxdb/v2.3/api-guide/tutorials/_index.md

influxdb/v2.3/tags: [api]
---

Follow step-by-step tutorials to build an Internet-of-Things (IoT) application with InfluxData client libraries and your favorite framework or language.
InfluxData and the user community maintain client libraries for developers who want to take advantage of:

- Idioms for InfluxDB requests, responses, and errors.
- Common patterns in a familiar programming language.
- Faster development and less boilerplate code.

> These tutorials walk through using the InfluxDB API and
> client libraries to build a modern application as you learn the following:

Reviewer: ```suggestion
In these tutorials, you'll use the InfluxDB API and
client libraries to build a modern application, and learn the following:
```
